pipeline-notebooks/pipeline-general.ipynb

```diff
@@ -540,20 +550,27 @@
    "outputs": [],
    "source": [
     "# pipeline-api\n",
-    "def pipeline_api(file, filename='', response_type=\"application/json\"):\n",
+    "\n",
+    "def pipeline_api(file, filename='', file_content_type=None, response_type=\"application/json\"):\n",
     "    # NOTE(robinson) - This is a hacky solution due to\n",
     "    # limitations in the SpooledTemporaryFile wrapper.\n",
-    "    # Specifically, it does't have a `seekable` attribute,\n",
+    "    # Specifically, it doesn't have a `seekable` attribute,\n",
     "    # which is required for .pptx and .docx. See below\n",
     "    # the link below\n",
     "    # ref: https://stackoverflow.com/questions/47160211\n",
     "    # /why-doesnt-tempfile-spooledtemporaryfile-implement-readable-writable-seekable\n",
     "    with tempfile.TemporaryDirectory() as tmpdir:\n",
-    "        _filename = os.path.join(tmpdir, filename.split('/')[-1])\n",
-    "        with open(_filename, \"wb\") as f:\n",
+    "        file_filename = os.path.join(tmpdir, filename.split('/')[-1])\n",
+    "        with open(file_filename, \"wb\") as f:\n",
```

Comment:
not strictly related to this PR, but the hack to rewrite the posted `file` object to another file (which is non ideal for a couple of reasons, including performance) seems like it is only needed for `.pptx` and `.docx` per Robinson's comment. can we keep it to that case? (or address in other PR... just did not want to lose track of this)